File: Java Labs/Patterns On Java/src/com/pattern/composite/ClothingCollection.java
package com.pattern.composite;

import java.util.ArrayList;
import java.util.List;

public class ClothingCollection implements ClothingItem {
    private List<ClothingItem> items = new ArrayList<>();

    public void addClothingItem(ClothingItem item) {
        items.add(item);
    }

    public void removeClothingItem(ClothingItem item) {
        items.remove(item);
    }

    @Override
    public void displayDetails() {
        System.out.println("Clothing Collection:");
        for (ClothingItem item : items) {
            item.displayDetails();
        }
    }
}

File: Python Labs/com/pattern/composite/ClothingCollection.py
from com.pattern.composite.ClothingItem import ClothingItem


class ClothingCollection(ClothingItem):
    def __init__(self):
        self.items = []

    def add_clothing_item(self, item):
        self.items.append(item)

    def remove_clothing_item(self, item):
        self.items.remove(item)

    def display_details(self):
        print("Clothing Collection:")
        for item in self.items:
            item.display_details()